xdg-ninja.sh

```diff
@@ -30,58 +28,115 @@ fi
 
 unalias -a
 
-HELPSTRING="""\
+#Set ansi color code variables
+set_colors() {
+    case $COLOR in
+        never)
+            DECODER="cat"
+            return
+        ;;
+        auto)
+            if [ ! -t 1 ] || [ $NO_COLOR ];then # Check if used in a pipe or if the NO_COLOR env variable is set.
+                DECODER="cat" 
+                return
+            fi
+        ;;
+    esac
+    ANSI_BEGIN="\033["
+    ANSI_END="m"
+
+    ANSI_CLEAR=";0"
+    ANSI_BOLD=";1"
+    ANSI_ITALLIC=";3"
+
+    ANSI_RED=";31"
+    ANSI_GREEN=";32"
+    ANSI_YELLOW=";33"
+    ANSI_CYAN=";36"
+    ANSI_WHITE=";37"
+
+    ANSI_BACKGROUND_PURPLE=";45"
+}
 
+help() {
+    set_colors
+    HELPSTRING="""\
 
-    \033[37;45;1mxdg-ninja\033[0m
 
-    \033[1;3mCheck your \$HOME for unwanted files.\033[1;0m
+        ${ANSI_BEGIN}${ANSI_BOLD}${ANSI_WHITE}${ANSI_BACKGROUND_PURPLE}${ANSI_END}xdg-ninja${ANSI_BEGIN}${ANSI_CLEAR}${ANSI_END}
```

Comment:
That escape code composition looks really ugly and litters the strings. Is it really needed there? What about just defining all escape codes used (like `BOLD_ITALIC`, `CLEAR_BOLD`, `CLEAR`, `CYAN_BOLD`, etc.) however (with composition or without) first and using those instead